docs/source/pages/using.rst

```diff
@@ -283,10 +303,12 @@ To run the analysis against that SBOM, run this command:
 
 .. code-block:: shell
 
-  ./run_macaron.sh analyze -purl pkg:maven/org.apache.maven/maven@3.9.7?type=pom -sbom <path_to_sbom>
+  ./run_macaron.sh analyze -purl pkg:maven/org.apache.maven/maven@3.9.7?type=pom -sbom <path_to_sbom> --deps-depth=1
```

Comment:
Should we instead use `--deps-depth=inf` for the SBOM examples? I doesn't really matter either way, but I would think that probably most people who want to run over an SBOM want to run over everything.

Reply:
Yep. I don't see why not switch to `--deps-depth=inf`. 
The reason I chose `--deps-depth=1` is because in this section originally no `recursive` option is set in `defaults.ini`, which means that this example was configured to analyze direct dependencies.

Reply:
Fixed in https://github.com/oracle/macaron/pull/840/commits/f402af08f16013b234f4b5d5277464d318002cdc